config/purposes/bespoke.yml

```diff
@@ -31,6 +31,10 @@ LBB Lib Pool Stock:
   :creator_class: LabwareCreators::PooledTubesBySubmission
   :presenter_class: Presenters::SimpleTubePresenter
   :default_printer_type: :tube
+  :file_links:
+    - name: 'Download MBRAVE file'
```

Comment:
I think it more accurately it is called MBRAVE UMI file (I think there are 3 MBrave input files altogether, this is one of them, there is also the sequencing data file and some sort of reference file)

Reply:
My mistake, this is from when we were coding this before having the pipeline

Reply:
This section is in config/purposes/bioscan.yml , under `LBSN-9216 Lib PCR Pool XP` purpose now.